hbase_sql/src/main/java/com/splicemachine/lifecycle/RegionServerLifecycle.java

```diff
@@ -64,7 +64,7 @@ public void distributedStart() throws IOException{
                             .newBuilder(spliceInit)
                             .setColumnFamily(
                                     ColumnFamilyDescriptorBuilder.newBuilder(Bytes.toBytes("FOO")).build()
-                            ).build();
+                            ).setPriority(1000).build();
```

Comment:
Should we really use such a high number? If I understand correctly, hbase's own system tables use 200, so I don't think we should go above that.

Reply:
SPLICE_INIT table should not be created, so no need to change its priority.